api/src/test/java/org/openmrs/module/ModuleFileParserUnitTest.java

```diff
@@ -98,22 +97,19 @@ public void after() {
 	@Test
 	public void moduleFileParserConstruction_shouldFailIfGivenNull() {
 
-		expectedException.expect(NullPointerException.class);
-		expectedException.expectMessage("messageSourceService must not be null");
-
-		new ModuleFileParser((MessageSourceService) null);
+		NullPointerException exception=assertThrows(NullPointerException.class, () -> new ModuleFileParser((MessageSourceService) null));
```

Comment:
```suggestion
		NullPointerException exception= assertThrows(NullPointerException.class, () -> new ModuleFileParser((MessageSourceService) null));
```

please pay attention to the whitespaces!